src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -132,21 +132,24 @@ private BoundInterpolatedString BindUnconvertedInterpolatedStringToString(BoundU
         {
             // We have 4 possible lowering strategies, dependent on the contents of the string, in this order:
```

Comment:
We have _5_ possible [...]

Reply:
The 2 versions of concat are really still one strategy.